wallet/txsizes/size.go

```diff
@@ -125,6 +156,14 @@ const (
 	//   - 1 wu compact int encoding value 33
 	//   - 33 wu serialized compressed pubkey
 	RedeemP2WPKHInputWitnessWeight = 1 + 1 + 73 + 1 + 33
+
+	// RedeemP2TRInputWitnessWeight is the worst case weight of
+	// a witness for spending P2TR outputs. It
+	// is calculated as:
+	//
+	//   - 1 wu compact int encoding value 65
+	//   - 64 wu BIP-340 schnorr signature + 1 wu sighash
+	RedeemP2TRInputWitnessWeight = 1 + 65
```

Comment:
FWIW, this'll be _slightly_ off in the case that sighash default is used (implicit alias for sighash all).

Reply:
Yeah, all of them said the worst case scenario so I included the sighash byte